components/basic/SvgMaker.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+
+const SvgMaker = ({ svgPath }) => {
+  const [hovered, setHovered] = useState(false);
+
+
+
+  return (
+    <svg
+      className={`hover:tw-animate-bounce ${
+        hovered ? 'tw-text-[#00C9A7] hover:tw-bg-primary-200' : 'tw-text-white'
+      } tw-text-3xl tw-bg-primary-100 tw-p-8 tw-rounded-full`}
+      onMouseOver={(e) => (e.currentTarget.style.fill = '#00C9A7')} // Green color on hover
```

Comment:
Don't use green color on hover, please use our theme/primary colors.
See it in tailwind config.

Reply:
Done lol 🤣